vortex-array/src/array/varbinview/mod.rs

```diff
@@ -106,7 +105,9 @@ impl Debug for BinaryView {
     }
 }
 
-pub const VIEW_SIZE: usize = mem::size_of::<BinaryView>();
+// this is a hack; views are 16 bytes with 8-byte alignment
```

Comment:
Not sure what kind of hack you're referring to. You can link to c++ code definition to show that this is the same as theirs